Netkan/Transformers/OptimusPrimeTransformer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using CKAN.NetKAN.Model;
+using log4net;
+using Newtonsoft.Json.Linq;
+
+namespace CKAN.NetKAN.Transformers
+{
+    internal sealed class OptimusPrimeTransformer : ITransformer
+    {
+        private static readonly ILog Log = LogManager.GetLogger(typeof(OptimusPrimeTransformer));
+
+        public Metadata Transform(Metadata metadata)
+        {
+            var json = metadata.Json();
+
+            JToken optimusPrime;
+            if (json.TryGetValue("x_netkan_optimus_prime", out optimusPrime) && (bool)optimusPrime)
+            {
+                Log.Info("Autobots roll out!");
```

Comment:
Oh my goodness,  *I* was going to make this joke somewhere when doing code review! You beat me to it! :)